File: server/models/posts.py
from mongoengine import Document, StringField, IntField, ImageField
class Post(Document):
    date = StringField(required=True)
    timestamp = StringField(required=True)
    user = StringField(required=True)
    tags = StringField()
    title = StringField(required=True)
    text = StringField()
    image = ImageField()
    firm = StringField()
    likes = IntField()

    meta = {'collection': 'posts'}

File: server/models/posts.js
const mongoose = require('mongoose');

const postSchema = new mongoose.Schema({
    date: {
        type: String,
        required: true
    },
    timestamp: {
        type: String,
        required: true
    },
    user: {
        type: String,
        required: true
    },
    tags: String,
    title: {
        type: String,
        required: true
    },
    text: String,
    image: {
        data: Buffer,
        contentType: String
    },
    firm: String,
    likes: Number,
}, {collection: "posts"});

module.exports = mongoose.model('posts', postSchema);